c2cgeoportal/scaffolds/create/deploy/hooks/pre-restore-database.mako

```diff
@@ -11,23 +11,26 @@ DATABASES=$@
 # databases / tables restore.
 sudo apache2ctl stop
 
+if psql -lqt | cut -d \| -f 1 | grep -qw $DATABASES; then
```

Comment:
we dont do any ´sudo -u postgresql´ in the other command either, why would we ?

if you know how to make the command line shorter, please tell me

I can remove the q option if you dont like it

I can change the --word-regexp